### PR TITLE
Grid added to UI to show pending registrations

### PR DESCRIPTION
**Pull request by sjawaji**
Namaskaram @thulasidhar anna,

First attempt to try to take some load off you.  Please review this change.

Additional Grid will be display under registrations Grid that will list pending registrations.

Pranam
Siva

### CODE REVIEW

File: src/main/webapp/js/controllers/programInfoCtrl.js [hunk 1]
```diff
@@ -340,12 +340,16 @@ angular.module('app')
                 backend.api("register.query", { programId: programId, updateCheckinStatus: true })
                     .then(function (resp) {
                         if (resp.statusCode == 'SUCCESS') {
+                            $scope.registrationsPending = _.filter(resp.object, function(reg){ return (reg.status != 'REGISTRATION_COMPLETE' && reg.status != 'CANCELLED' ); });
```
Comment (thulasidhar) [inline src/main/webapp/js/controllers/programInfoCtrl.js]: The check can be for AUTHORIZATION_PENDING

File: src/main/webapp/js/controllers/programInfoCtrl.js [hunk 2]
```diff
@@ -396,6 +400,10 @@ angular.module('app')
                 { field: 'registeredMs', width: '7%', minWidth: 70, cellClass: 'text-center' }
             ];
 
+            $scope.regPendingColDefs = $scope.regColDefs.slice(2, 6);
```
Comment (thulasidhar) [inline src/main/webapp/js/controllers/programInfoCtrl.js]: Better to make this explicit for readability purposes and also to keep it decoupled from the other options object.

File: src/main/webapp/templates/program-info.html [hunk 3]
```diff
@@ -50,6 +50,16 @@ <h5 class="bold" ng-show="showCollection">Collected SGD: {{program.regSummary.st
         <div ui-grid="regGridOptions" ui-grid-pagination ui-grid-exporter ui-grid-resize-columns class="ui-grid-style vpad"
              ng-show="registrations"></div>
     </div>
+    <div class="row" ng-if="program.regSummary">
```
Comment (thulasidhar) [inline src/main/webapp/templates/program-info.html]: Can we show the pending regs similar to cancellations? via a modal...
Or do you want to keep it like this, so that we can easily export into csv?

File: src/main/webapp/js/controllers/programInfoCtrl.js (hunk 1 above)
Comment (sjawaji) [inline src/main/webapp/js/controllers/programInfoCtrl.js]: I noticed additional statuses like these - ERROR_CREATING_PAYMENT_URL.  So, wanted to pull everything minus those two statuses.

File: src/main/webapp/templates/program-info.html (hunk 3 above)
Comment (sjawaji) [inline src/main/webapp/templates/program-info.html]: Would like to have the option to export.  Hope grid is supported in modal's.  I will explore this option of showing the Grid in the modal.

File: src/main/webapp/js/controllers/programInfoCtrl.js (hunk 2 above)
Comment (sjawaji) [inline src/main/webapp/js/controllers/programInfoCtrl.js]: Will fix it and will send another pull request.
